scripts/docker/alpine.3.6/Dockerfile

```diff
@@ -4,7 +4,7 @@
 #
 
 # Dockerfile that creates a container suitable to build dotnet-cli
-FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-20171119021156
+FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6-3e800f1-20190501005341
```

Comment:
This is using the wrong repo.  `microsoft/dotnet-buildtools-prereqs` has been out of support for a long time now although all images that were published there remain there.  You will need to update the `mcr.microsoft.com/dotnet-buildtools/prereqs` repository and everything should work.

Reply:
Thanks!